api/src/pcapi/scripts/subscription/dms/handle_inactive_dms_applications.py

```diff
@@ -89,10 +89,12 @@ def _mark_without_continuation_a_draft_application(dms_application: dms_models.D
     logger.info("[DMS] Marked application %s without continuation", dms_application.number)
 
 
-def _mark_cancel_dms_fraud_check(application_number: int) -> None:
+def _mark_cancel_dms_fraud_check(application_number: int, email: str) -> None:
     try:
-        fraud_check = fraud_models.BeneficiaryFraudCheck.query.filter_by(
-            type=fraud_models.FraudCheckType.DMS, thirdPartyId=str(application_number)
+        fraud_check = fraud_models.BeneficiaryFraudCheck.query.filter(
+            fraud_models.BeneficiaryFraudCheck.type == fraud_models.FraudCheckType.DMS,
+            fraud_models.BeneficiaryFraudCheck.thirdPartyId == str(application_number),
+            fraud_models.BeneficiaryFraudCheck.resultContent.contains({"email": email}),
```

Comment:
il faut peut-être faire `BeneficiaryFraudCheck.resultContent is not None and ...` pour pas avoir d'erreur si jamais resultContent est null, non ?

Reply:
sinon une autre possibilité est de faire une jointure sur la table user